src/poojap/AreaPerimeter.java

```diff
@@ -0,0 +1,30 @@
+package poojap;
+
+public class AreaPerimeter {
+	
+	float pi= 3.1416f;
+	    
+	void calculateArea(int lenght, int width,int radius) {
+		
+		double rectangleArea = lenght*width; 
+		System.out.println("Area of Rectanlge is " +rectangleArea);
```

Comment:
Here you should create separate method for the circle area and triangle area, here it looks if i am asking for circle area then you are giving me triangle area free.